glossary.md

```diff
@@ -6,28 +6,97 @@
 * `Library`: The collection of all `responses`.
 * `Registry`: The collection of available `survey objects` for use in `responses`.
 * `Analysis`: The process of converting many survey responses to a Sankey diagram by an
-  `Analyst` interacting with the `Library`. Surveys are searched by title and `tags`.
+  `Analyst` interacting with the `Library`. Surveys are searched by title, `tags`, and/or `object metadata`.
 
 
-## Response Object Registry
+## Registered Survey Object Registry
 
 A list of available `response objects`. Drives consistency in naming across `surveys` to
 support `analysis` and also to ease data entry for `Experts`, over time.
 
-Experts _may not_ add new `applications` to the `registry`, but _may_ add new `data
-products` and `observing systems` to the `registry`.
+Experts may add new `applications` `data products` and `observing systems` to the `registry`.
```

Comment:
```suggestion
Experts may add new `applications`, `data products`, and `observing systems` to the `registry`.
```

Reply:
Do I need to do anything to resolve/approve these changes?